js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializeRunHistoryTable.tsx

```diff
@@ -32,9 +35,7 @@ export const AutomaterializeRunHistoryTable = ({
       return data.pipelineRunsOrError.results;
     },
     variables: {
-      filter: {
-        tags: [{key: 'dagster/auto_materialize', value: 'true'}],
-      },
+      filter: queryFilter,
```

Comment:
Make this a prop so that it can be used by the current Auto-materialization overview page and the new auto-materialize sensor page -- they use slightly different tag filters in the same runs query.

Reply:
Ack, will fix.